apps/els_core/src/els_utils.erl

```diff
@@ -141,6 +144,23 @@ lookup_document(Uri) ->
       end
   end.
 
+%% @doc Convert path to an 'include' POI id
+-spec include_id(file:filename_all()) -> string().
+include_id(Path) ->
+  els_utils:to_list(filename:basename(Path)).
```

Comment:
Do we need to convert to list(char()) here? filename:basename() should return a string

Reply:
good question, I hit this while testing
the new code here https://github.com/erlang-ls/erlang_ls/pull/1001/files#diff-2ae95695a32108b6ba4383bafbc080cea105dbd9ba88c6a45474444ef3773471R71
calls `include_id/1` with a binary from `els_uri:path/1` and `filename:basename/1` returns a binary if called with a binary. (Similarly `filename:join/1` returns a binary if any of the components are binary)